Document the "Propose changelog after file dnd" option (#390)
Enalean/tuleap-documentation-en: languages/en/administration-guide/application-management/plugins/document.rst

The first limit **forbids** users from downloading folders above this size. The
second limit shows a warning to users but still allows downloading folders
above this size.

History enforcement
```````````````````

.. figure:: ../../../images/screenshots/document/siteadmin_history_enforcement.png
  :alt: Screenshot of the "History enforcement" tab in Site administration

> By default, when a user drops a new version on a file to update it, he/she is not asked to fill up anything.

Reviewer: ```suggestion
By default, when a user drops a new version on a file to update it, they are not asked to fill up anything.
```

> In order to encourage people to fill up the version changelog of files, you can enable the option "Propose changelog after drag and drop".

Reviewer: ```suggestion
In order to encourage people to fill up the version changelog of files, site administrators can enable the option "Propose changelog after drag and drop".
```

> Each time a new version will be uploaded using dn'd, a changelog modal will be displayed beforehand.

Reviewer: ```suggestion
Each time a new version will be uploaded using drag'n drop, a changelog modal will be displayed beforehand.
```
